content/docs/ref/index.md

```diff
@@ -1,5 +1,18 @@
 # Command Reference
 
+CML provides a number of commands to help package the outputs of ML workflows
+into a CML report, which may include numeric data or model performance
+visualizations. Let's look at the typical sequence (after
+[configuration](/doc/user-guide)):
```

Comment:
Moved here from https://cml.dev/doc/usage#cml-commands

Reply:
> can we move it back?

@casperdcl don't we need some content in the cmd ref index though? Currently it's empty except for a **Generic Options** section. Seems a little bare. Imagine a user lands in this page as their entry point to the docs site (per analytics this does happen sometimes) or as one of the first pages visited. It should give some general info on `cml` commands as a whole, link to GS or Usage, etc., I think.

Granted, this could be addressed separately and/or we can repeat the text above in you prefer to also keep this general guidance in the Usage (now UG index) page. Lmk

Reply:
please revert